### PR TITLE
Add Maya version and build type, minor tweaks

### PR DESCRIPTION
**Pull request by luisbarrancos**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -40,10 +40,27 @@ set (CMAKE_MODULE_PATH
...
+if (NOT CMAKE_BUILD_TYPE)
+    set (CMAKE_BUILD_TYPE Debug
+         CACHE STRING "Choose the type of build, options are: ${CMAKE_CONFIGURATION_TYPES}" FORCE)
+    set_property (CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS ${CMAKE_CONFIGURATION_TYPES})
+endif ()  
 
 #--------------------------------------------------------------------------------------------------
 # Version.
-#--------------------------------------------------------------------------------------------------
+#---------------------------------------------------------o-----------------------------------------
```
Comment (dictoon) [inline CMakeLists.txt]: ? :)

File: src/appleseedmaya/CMakeLists.txt [hunk 2]
```diff
@@ -159,8 +155,14 @@ target_link_libraries (appleseedMaya
...
+                       DEBUG_POSTFIX "_Debug"
+                       RELEASE_POSTFIX "_Release"
+                       OUTPUT_NAME
+                       "${MAYASEED_PLUGIN_NAME}-${MAYA_API_VERSION_SHORT}"
+                       PREFIX ""
+                       )
 
 if (${CMAKE_SYSTEM_NAME} MATCHES "Windows")
     set_target_properties (appleseedMaya PROPERTIES SUFFIX ".mll")
-endif ()
```
Comment (dictoon) [inline src/appleseedmaya/CMakeLists.txt]: Revert this change.

**Review by dictoon — CHANGES_REQUESTED** [review]

File: src/appleseedmaya/CMakeLists.txt [hunk 3]
```diff
@@ -159,8 +155,14 @@ target_link_libraries (appleseedMaya
     ${PYTHON_LIBRARIES}
 )
 
-set_target_properties (appleseedMaya PROPERTIES PREFIX "")
+set_target_properties (${MAYASEED_PLUGIN_NAME} PROPERTIES
```
Comment (est77) [inline src/appleseedmaya/CMakeLists.txt]: I prefer if we don't do this. It's important when scripting that the plugin has a fixed name.

**Review by est77 — CHANGES_REQUESTED** [review]

File: src/appleseedmaya/CMakeLists.txt (hunk 3 above)
Comment (luisbarrancos) [reply src/appleseedmaya/CMakeLists.txt]: @est77 yeah, reverting that.

**Review by dictoon — APPROVED** [review]

**Comment by dictoon** [comment]
Looking good, please squash.
